app/jenkins-for-jira-ui/src/components/JenkinsSetup/JenkinsSetup.tsx

```diff
@@ -214,10 +261,17 @@ const JenkinsSetup = (): JSX.Element => {
 	const [serverName, setServerName] = useState('');
 	const [showMyJenkinsAdmin, setShowMyJenkinsAdmin] = useState(false);
 	const [showIAmTheJenkinsAdmin, setShowIAmTheJenkinsAdmin] = useState(false);
+	const [isCopyAdminSiteName, setIsCopyAdminSiteName] = useState(false);
+	const [isCopyAdminWebhook, setIsCopyAdminWebhook] = useState(false);
+	const [isCopyAdminSecret, setIsCopyAdminSecret] = useState(false);
+	const [isCopyNonAdminWebhook, setIsCopyNonAdminWebhook] = useState(false);
+	const [isCopyNonAdminSecret, setIsCopyNonAdminSecret] = useState(false);
 	const [webhookUrl, setWebhookUrl] = useState('');
 	const [secret, setSecret] = useState<string>('');
 	const [siteName, setSiteName] = useState<string>('');
 	const [globalPageUrl, setGlobalPageUrl] = useState<string>('');
+	const [primaryCopyButtonName, setPrimaryCopyButtonName] =
+	useState<CopyButtonNameEnum>(CopyButtonNameEnum.NonAdminWebhook);
```

Comment:
/nit indentation

Reply:
Eslint would complain if I add indentation. I think eslint want this style even I prefer adding an indentation indeed.